.gitlab-ci.yml

```diff
@@ -25,4 +25,26 @@ limani:
     refs:
       - main
     changes:
-      - src/**/*
\ No newline at end of file
+      - src/**/*
+
+blackduck:
+  stage: scan
+  image: openjdk:latest
```

Comment:
Can you use a specific version of openjdk and one that is based of debian bullseye

Reply:
Done